### PR TITLE
Edusign app/sp updates for latest version

### PR DESCRIPTION
**Pull request by kushaldas**
This has tested in `edusign` test servers.

### CODE REVIEW

File: manifests/edusign/app.pp [hunk 1]
```diff
@@ -32,14 +32,16 @@
       image    => 'docker.sunet.se/edusign-sp',
       imagetag => $version,
       hostname => $facts['networking']['fqdn'],
-      volumes  => ['/var/log:/var/log','/etc/ssl:/etc/ssl','/etc/dehydrated:/etc/dehydrated','/etc/metadata:/etc/metadata:ro','/etc/edusign:/etc/edusign:ro'],
+      volumes  => ['/var/log:/var/log','/etc/ssl:/etc/ssl','/etc/dehydrated:/etc/dehydrated','/etc/metadata:/etc/metadata:ro','/etc/edusign:/etc/edusign:ro', "/var/run/md-signer2.crt:/etc/shibboleth/md-signer2.crt:ro"],
```
[the remark this answers is not in the document]
Comment (SpaceFarmer) [reply manifests/edusign/app.pp:35]: If we could fix the puppet lint suggestion to use single quotes on the i think that would be good:
`"/var/run/md-signer2.crt:/etc/shibboleth/md-signer2.crt:ro"`

**Review by SpaceFarmer — CHANGES_REQUESTED** [review]
LGTM if we fix the puppet-lint suggestion about changing to singel quotes.

File: manifests/edusign/app.pp (hunk 1 above)
[the remark this answers is not in the document]
Comment (kushaldas) [reply manifests/edusign/app.pp]: This is fixed now.

**Review by SpaceFarmer — APPROVED** [review]

**Review by SpaceFarmer — APPROVED** [review]
LGTM

**Pull request merged**